src/transaction_signature_options/sign_with_access_key_file/mod.rs

```diff
@@ -65,6 +65,30 @@ impl SignAccessKeyFileContext {
         let signature = account_json
             .private_key
             .sign(unsigned_transaction.get_hash_and_size().0.as_ref());
+
+        if let Some(_) = &network_config.meta_transaction_relayer_url {
+            let max_block_height = rpc_query_response.block_height + 1000; //XXX
+
+            let signed_delegate_action = super::get_signed_delegate_action(
+                unsigned_transaction,
```

Comment:
Do not construct `unsigned_transaction` when it is not necessary, use `prepopulated_transaction` instead.

Reply:
Done